src/deluge/gui/views/view.cpp

```diff
@@ -203,6 +204,22 @@ ActionResult View::buttonAction(deluge::hid::Button b, bool on, bool inCardRouti
 		}
 	}
 
+	// Load button for MIDI learn
```

Comment:
per my comment above, not sure we need this shortcut given that all other global midi commands don't have shortcuts like this. It's a learn once type thing. I really don't think we need a shortcut for it.